samples/bluetooth/channel_sounding/README.rst

```diff
@@ -0,0 +1,56 @@
+.. zephyr:code-sample:: ble_cs
+   :name: Channel Sounding
+   :relevant-api: bt_gap bluetooth
+
+   Use Channel Sounding functionality.
+
+
+Bluetooth: Channel Sounding
+###########################
+
+Overview
+********
+
+These samples demonstrates how to use the Bluetooth Channel Sounding feature.
+
+The CS Test sample shows how to us the CS test command to override randomization of certain channel
+sounding parameters, experiment with different configurations, or evaluate the RF medium. It can
+be found under :zephyr_file:`samples/bluetooth/channel_sounding/cs_test`.
+
+The connected CS sample shows how to set up regular channel sounding procedures on a connection
+between two devices.
+It can be found under :zephyr_file:`samples/bluetooth/channel_sounding/connected_cs`
+
```

Comment:
Would it be possible to add a simple message sequence chart visualizing how the samples are obtained and exchanged?

Reply:
added references to some MSCs in the spec. The rest is just a simple GATT write